### DISCUSSION TITLE
How are map color texture work?

### DISCUSSION THREAD

**Discussion by TheNotEasy** [efroemling/ballistica#619] [Q&A] [answered]
For example, here is the texture of the bridgit map:
![bridgitLevelColor](https://github.com/efroemling/ballistica/assets/82557920/3635afc9-a972-4506-965f-1895f7605040)

If this is a render of a model, what camera angle and distance should it be, and should the texture have a 1:1 ratio?
I want to create my own map

P.S I'm using bob_plugin.py with blender 2.77

**Comment by JoseANG3L** [comment]
In my experience, I create the map, add the texture, add lighting, use the cameras at points where the map will be visible and render it.
I put that final render in Photoshop in a texture with a 1:1 size, in the end I use that texture in the map model so that it fits with the final render.

**Reply by TheNotEasy** [reply]
Does this mean that you can put a camera in a blender and the game will use it position in game? If not, how to know where to place camera?

**Reply by JoseANG3L** [reply, marked as answer]
No, the camera in Blender is to render the camera view, the map, the game camera is guided from the boxes and points that you define, most maps are in the center of these boxes.
You can see the example of The Pad map, its defs, it's called the_pad.py, You can also import that file into Blender, it's the last option, it's below importing collisions.
